argus/client/generic/cli.py

```diff
@@ -39,8 +41,23 @@ def finish_run(api_key: str, base_url: str, id: str, status: str, scylla_version
     client.finalize_generic_run(run_id=id, status=status, scylla_version=scylla_version)
 
 
+@click.command("trigger-jobs")
+@click.option("--api-key", help="Argus API key for authorization", required=True)
+@click.option("--base-url", default="https://argus.scylladb.com", help="Base URL for argus instance")
+@click.option("--job-info-file", required=True, help="JSON file with trigger information (see detailed docs)")
+def trigger_jobs(api_key: str, base_url: str, job_info_file: str):
```

Comment:
can we make job-info-file consist only jenkins params and things like version, plan_id, release be specified in cli?
This way maybe we could commit job-info-files and reuse across releases?

Reply:
Hmm, good idea, will do it that way.

Reply:
Done